tests/test_app.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from dotenv import find_dotenv, load_dotenv
+import requests
+import app
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    # Create the new app.
+    test_app = app.create_app()
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+class MockCardsResponse:
+    def json():
+        return [
+            {
+                "id": "5ef7a4f991ad48544d0f2a53",
+                "idList": "todo_list_id",
+                "name": "This is a to do item",
+                "desc": "Description for to do item",
+                "dateLastActivity": "2020-07-30T12:52:06.278Z"
+            },
+            {
+                "id": "5ef7a4f991ad48544d0f2a54",
+                "idList": "doing_list_id",
+                "name": "This item is currently in progress",
+                "desc": "Description for item in progress",
+                "dateLastActivity": "2020-07-30T12:52:06.278Z"
+            },
+            {
+                "id": "5ef7a4f991ad48544d0f2a55",
+                "idList": "done_list_id",
+                "name": "This item is done",
+                "desc": "Description for done item",
+                "dateLastActivity": "2020-07-30T12:52:06.278Z"
+            },
+        ]
+
+@pytest.fixture()
+def mock_get_requests(monkeypatch):
+    def mock_get(*args, **kwargs):
+        return MockCardsResponse
```

Comment:
You could make this check args and only return MockCardsResponse when args[1] is the correct URL.

It would mean the test only passes when the app uses the correct URL and would also let you mock multiple endpoints simultaneously, if you have more tests/more complicated tests.

Reply:
Thanks! I'll have a go at this.